src/lib/classes/lsp3-universal-profile.spec.ts

```diff
@@ -181,7 +152,83 @@ describe('LSP3UniversalProfile', () => {
 
       const [result] = await universalProfile.connect(signers[0]).callStatic.getData([key]);
       const checkedsumResult = ethers.utils.getAddress(result);
-      expect(checkedsumResult).toEqual(secondController);
+      expect(checkedsumResult).toEqual(secondControllerAddress);
+    });
+
+    it.only('first account should be able to setData', async () => {
+      const abi = await universalProfile.populateTransaction.setData(
+        ['0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5'],
+        ['0xbeefbeef']
+      );
+
+      const result = await keyManager.connect(signers[0]).execute(abi.data);
+
+      expect(result).toBeTruthy();
+
+      const data = await universalProfile.getData([
+        '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
+      ]);
+
+      expect(data).toEqual(['0xbeefbeef']);
+    });
+    it.only('second account should be able to setData', async () => {
+      const abi = await universalProfile.populateTransaction.setData(
+        ['0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5'],
+        ['0xcafecafe']
+      );
+
+      const result = await keyManager.connect(signers[1]).execute(abi.data);
+
+      expect(result).toBeTruthy();
+
+      const data = await universalProfile.getData([
+        '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
+      ]);
+
+      expect(data).toEqual(['0xcafecafe']);
```

Comment:
you can put `0xcafecafe` in a variable and reuse that variable at line 177 and 188.

So when you edit the input you also edit the expect. No need to update 2 lines + it makes it easier to see what value you are checking.